src/style.css

```diff
@@ -0,0 +1,138 @@
+html {
+  font-family: "Roboto", sans-serif;
+}
+
+body {
+  margin: 0;
+  --main-color: #000;
+  --secondary-color: #777;
+  --moyo-color: #00acdc;
+}
+
+h2,
+h3,
+p {
+  margin: 0;
+}
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  box-sizing: border-box;
+  padding: 32px 16px 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__img {
+  width: 160px;
+  height: 134px;
+  margin-bottom: 40px;
+}
```

Comment:
When you remove flex to put img in center give her d: block, and margin l,r: auto